packages/injected/src/clock.ts

```diff
@@ -192,38 +202,63 @@ export class ClockController {
   }
 
   private _updateRealTimeTimer() {
-    if (!this._realTime) {
```

Comment:
Since this method is called unconditionally from `addTimer()`, we must still check `this._realTime`.